tomviz/vtkNonOrthoImagePlaneWidget.h

```diff
@@ -498,6 +516,10 @@ class vtkNonOrthoImagePlaneWidget : public vtkPolyDataSourceWidget
   void GenerateArrow(); // generate the default arrow
   void UpdateArrowSize(); // update the arrow to be visible based on camera pos
 
+  bool PickPointOnSlice(
+    int displayPos[2],
```

Comment:
Indentation looks odd, I think vtkVector2i and vtkVector3d would be better to use here, and in the API.